app/models/intake.rb

```diff
@@ -568,4 +576,13 @@ def new_dependent_token
     verifier = ActiveSupport::MessageVerifier.new(Rails.application.secret_key_base)
     verifier.generate(SecureRandom.base36(24))
   end
+
+  def compute_canonical_email_address
+    if email_domain == 'gmail.com'
+      username, domain = email_address.split('@')
```

Comment:
In the unlikely event there's two `@` characters, do you want to handle that? I guess not because that's handled by our email validator. OK! Ignore this comment.